src/objects/obstacles/pipes.ts

```diff
@@ -88,8 +91,14 @@ export class Pipes extends BaseObject {
    * randomly generate new top and bottom pipes
    */
   private generatePipes() {
+
+    //Pipe appears in order
+    this.pipeIndex = (this.pipeIndex + 1) % CONFIG.PIPE_TYPES.length;
+    const selectedPipeType = CONFIG.PIPE_TYPES[this.pipeIndex];
```

Comment:
so this is a tad confusing to read, and ends up having the array start at 1 not 0
```
const pipes = [...Array(6).keys()].map((i) => `pipe${i}`);
let pipeIndex = 0;
for (const _ of [...Array(20).keys()]) {
  pipeIndex = (pipeIndex + 1) % pipes.length;
  console.log(pipeIndex);
}

node test.js
1
2
3
4
5
0
1
2
3
4
5
0
1
2
3
4
5
0
1
2
```

So what do you think of doing
```
const selectedPipeType = CONFIG.PIPE_TYPES[this.pipeIndex % CONFIG.PIPE_TYPES.length];
this.pipeIndex += 1
```

So it selected each one from the list, then moves onto the next one for next pipe?
Or personally, what about making the order of pipes random? so each playthrough you get different buildings?